### ISSUE TITLE
Getting two console warnings

### ISSUE THREAD

**Issue by anzzstuff** [fritz-c/react-image-lightbox#10]
Everything is working as expected, but I'm wondering why I'm getting these two warnings in chrome dev tools' console:

```
Warning: Unknown prop `portalId` on <div> tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop
    in div

Warning: Unknown prop `_radiumDidResolveStyles` on <div> tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop
    in div (created by StyleRoot)
    in StyleRoot (created by ReactImageLightbox)
    in div
```

**Comment by fritz-c** [comment]
Apologies for the inconvenience. Since React 15.2.0, React has started to throw warnings whenever unknown properties were assigned to DOM elements. I'm currently working on a branch that will resolve the top warning, but the bottom warning is dependent on [this issue for Radium](https://github.com/FormidableLabs/radium/issues/759) being resolved.

**Comment by fritz-c** [comment]
This is fixed in 3.0.0. Please let me know if there are any degradations on your upgrade (see the [release notes](https://github.com/fritz-c/react-image-lightbox/releases/tag/v3.0.0) for details)
